src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  background-color: $text-color--white;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  &__img {
+    align-self: center;
+
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-top: 40px;
+
+    font-size: $font-size--title;
+    font-weight: $font-weight--title;
+    line-height: $line-height--accent;
+    color: $text-color--main;
+  }
+
+  &__code {
+    margin-top: 4px;
+
+    line-height: 14px;
+    color: $text-color--secondary;
+  }
+
+  &__rating {
+    margin-top: 16px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__reviews {
+    line-height: 14px;
+
+    color: $text-color--main;
+  }
+
+  &__price {
+    margin-top: 24px;
+
+    display: flex;
+    justify-content: space-between;
+
+    line-height: $line-height--accent;
+
+    &-text {
+      margin: 0;
+
+      font-size: $font-size--title;
+      line-height: $line-height--accent;
+      color: $text-color--secondary;
+    }
+
+    &-amount {
+      margin: 0;
+
+      font-size: $font-size--accent;
+      font-weight: $font-weight--accent;
+      line-height: $line-height--accent;
+      color: $text-color--main;
+    }
+  }
```

Comment:
`-text` is not an element or modifier of `card__price`, so it's a wrong usage of nesting. Although it provides ability to store some class name (like variable), we shouldn't use it for class name combining (it's not clear if it's nested class, or just other class with name that includes this word)